python/docs/memory.md

```diff
@@ -405,12 +468,38 @@ _Source: [python/examples/memory/summarizeMemory.py](/python/examples/memory/sum
 To create your memory implementation, you must implement the `BaseMemory` class.
 
 <!-- embedme examples/memory/custom.py -->
-
+From [custom.py](/python/examples/memory/custom.py):
 ```py
-# Coming soon
+from typing import Any
+
+from beeai_framework.backend.message import Message
+from beeai_framework.errors import UnimplementedError
+from beeai_framework.memory import BaseMemory
+
+
+class MyMemory(BaseMemory):
+    @property
+    def messages(self) -> list[Message]:
+        raise UnimplementedError("Method not yet implemented.")
```

Comment:
#409 proposes a change to remove this exception. At the time of PR creation no components were using this custom exception
In an earlier review @Tomas2D suggested we just use Python's standard **NotImplementedError**
Review comments in above PR welcome!